src/lib/models/solc.ts

```diff
@@ -21,10 +21,14 @@ export type CompilerInput = {
   };
 };
 
+export type ImportContents = Record<string, { contents: string }>
```

Comment:
This looks functionally the same as the `ContractSources` type below.

Reply:
the type was missing here but imported in [src/routes/compiler/compiler.ts](https://github.com/OpenZeppelin/defender-deploy-plugin/pull/47/files/2f1014aaff78222ba509aeafac06f89848f88c57#diff-7c81eecd8a62fa5ef71a0f7226a9403cd3a1cb93198cd64bbb494897e5277df3)

Reply:
Oh good catch.  I realize we could simplify this a lot, see my comment in `src/routes/compiler/compiler.ts`.

In that case, we can just remove `ImportContents` since it is specific only to the second parameter of solc-js's compile function, which we no longer need.